.github/workflows/release-please.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow opens and updates a pull request with a new package version
+# based on code changes.
+
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: google-github-actions/release-please-action@v3.7.10
+        with:
+          release-type: simple
```

Comment:
According to the https://github.com/marketplace/actions/release-please-action#release-types-supported, `simple` release type requires `version.txt` and `CHANGELOG.md`, but I don't see the former in the repo:

![2023-10-04_16-14-40_CEST_screenshot](https://github.com/ButterCMS/buttercms-csharp/assets/16722326/d583f8e7-ee7e-4895-85eb-58673d87a99c)

Are you sure it will work as expected?

Reply:
@Toreno96  I use the release type "simple" because release-please lib does not support .NET yet. But it works since I included Release-As: 2.0.0 in the commit message to initialize versioning (it won't start from 1.0.0.). After merging this release, it iterates over versions as expected. I have tested that on my fork: 

- This is PR with releasing 2.0.0 version - https://github.com/vlnevyhosteny/buttercms-csharp/pull/6 . As you can see, it just generates a changelog.
- Then I tried to create some dummy changes to test it, and the correct PR was created - https://github.com/vlnevyhosteny/buttercms-csharp/pull/5 . As it was a feat, it iterated from 2.0.0 to 2.1.0.

So I can include version.txt, but it's not necessary.

Reply:
OK, I was actually curious what's that `Release-As: 2.0.0` in the PR description, didn't know it was for `release-please`. Nice trick!

Thanks for the detailed test.

> So I can include version.txt, but it's not necessary.

Not necessary indeed, let's leave it as is.